deployments/data102/hubploy.yaml

```diff
@@ -1,7 +1,7 @@
 images:
   images:
     - name: us-central1-docker.pkg.dev/ucb-datahub-2018/user-images/primary-user-image
```

Comment:
The name should be changed too. Most deployment image names are of the form `{deployment}-user-image`. They also go under images.image_name -- see data100 as an example.